openedx-unsupported/devstack
# Target changed to dev image for devstack for Studio

**Task description:**
https://github.com/openedx/edx-platform/issues/31737

**How to test**
- Go to devstack
- Make sure you have latest edx-platform repo in workspace
- Now, all set to up Studio
- Run `make dev.up.studio`

**Commands to test**
- Make sure `make dev.shell.studio` is running fine and opening shell
- Make sure `make dev.provision.lms` is running fine
- Make sure `make dev.logs.studio` is not throwing any errors, it should run fine and showing logs
- Now got http://localhost:18010/home/ and check is everything working fine
